frontend/src/components/Portfolios/PortfoliosComboBox/PortfoliosComboBox.jsx

```diff
@@ -26,20 +26,20 @@ export const PortfoliosComboBox = ({
     let newPortfolioOptions = [];
 
     if (portfolioOptions.length === 0 && isSuccess) {
-        newPortfolioOptions = data?.map((portfolio, index) => {
+        newPortfolioOptions = data?.map((portfolio) => {
             const portfolioOption = {
-                id: index + 1,
+                id: portfolio.id,
```

Comment:
- 🏆 nice work
- ensure other comboboxes don't suffer from index issue
- This is the [PR](https://github.com/HHS/OPRE-OPS/pull/3843/files) that introduced the bug.